src/pudl/analysis/ferc1_eia.py

```diff
@@ -1263,7 +809,7 @@ def _get_subtable(table_name):
         return r_eia_years[r_eia_years.record_id_ferc1.str.contains(f"{table_name}")]
 
     def _get_match_pct(df):
-        return round(len(df[df["record_id_eia"].notna()]) / len(df))
+        return len(df[df["record_id_eia"].notna()]) / len(df)
```

Comment:
This `round` was rounding to either 1 or 0 since the number it's rounding is between 0 and 1. You can get rid of this round entirely since the logged coverage below is rounding to .01 in the f string.

Reply:
Leaving this comment here since I can't comment on the actual line itself. 

In the below coverage stats, the coverage for each of the FERC tables is printing that coverage for all years of EIA data, not just the subset that was included in the `pudl_out` object that generated the plant parts list for this model run. So when the CI runs, the coverage numbers are going to be low because they were only ever looking at 5 years of data. But I guess the numbers are accurate, because with 5 years of data the coverage during all EIA working years will be low. It's just not a very helpful stat but maybe it doesn't really matter.

Reply:
oh good! i noticed that these outputs were all 0% when i moved it over and it was wigging me out but i hadn't investigated why yet.